docs/src/modify.md

```diff
@@ -28,7 +28,7 @@ The `rename` method allows the column name(s) to be changed:
 ```@repl
 using TimeSeries
 using MarketData
-rename(cl, "New Close")
-rename(cl, ["New Close"])
-rename(ohlc, ["New Open", "New High", "New Low", "New Close"])
+rename(cl, :Close′)
```

Comment:
Is the `'` here intentional?

Reply:
`′` (not `'`) is used several times in https://github.com/JuliaStats/TimeSeries.jl/blob/master/src/broadcast.jl

I was surprised by the use of such a character also which can't be found easily on my keyboard

Reply:
Those render exactly the same for me. Weird.

Reply:
Why not naming simply `:NewClose` (or `:Close2`)?

What are the english name of `′`, `'` and `

Reply:
See screenshot to see how it's rendering here (France)
![capture d ecran 2018-10-16 a 22 39 24](https://user-images.githubusercontent.com/5049737/47045979-697d8980-d194-11e8-9694-19e0b02f41ed.png)

Reply:
Not sure about the first one, the second is "apostrophe," and the third is "backtick" (assuming that's rendering for me properly).

Reply:
> Why not naming simply `:NewClose` (or `:Close2`)?

well, I personally love the unicode support in Julia.
I always write code (in my research project) like `x = 1; x\prime<tab> = x + 42`
or 
`x\vec<tab> ... `

```julia
julia> x = 1
1

julia> x′ = x + 42
43

julia> function ϕ()
        42
       end 
ϕ (generic function with 1 method)

julia> :Close₂                                                   
```

Reply:
Oh, I see. I was on Linux before when it rendered like an apostrophe for me, but on macOS they're distinct.

Reply:
@ararslan maybe you should open an issue on your Linux distribution bug tracker because that's quite problematic to have 2 differents characters rendered the same.